examples/scripts/cards.js

```diff
@@ -1,3 +1,19 @@
+// Description:
+//   a bot to test this adapter
+//
+// Dependencies:
+//
+// Configuration:
+//
+// Commands:
+//   hubot help - will display some help
+//   hubot card - Replies in the same thread as the received message
+//
+// Notes:
+//
```

Comment:
Thank you for adding the documentation! Feel free to remove the help command since it looks like calling `hubot help` will display this command list by default.

Reply:
Will do.